Doc/library/unittest.mock.rst

```diff
@@ -1092,6 +1092,8 @@ object::
       >>> mock.await_args_list
       [call('foo'), call('bar')]
 
+  .. versionadded:: 3.8
```

Comment:
I would suggest moving this to somewhere near the class definition signature at https://github.com/python/cpython/pull/17102/files#diff-4d545f72ef0b42afd6a20165e4374004L855 before the first paragraph of doc so that users can quickly know without scrolling down.

NEWS entry is not required for this since it's a documentation change. We can add "skip-news" label to keep CI happy.

Reply:
I've moved it to just ahead of the method definitions, following e.g. https://docs.python.org/3/library/contextlib.html#contextlib.ExitStack